### ISSUE TITLE
prefer-module should allow a list of files to ignore

### ISSUE THREAD

**Issue by Nibblesh** [sindresorhus/eslint-plugin-unicorn#1233]
config files generally do not use ESM format, e.g. webpack.config.js, lint-staged.config.js

prefer-module should have a whitelist function to list the config files you want to allow usage of `module.exports`

**Comment by fisker** [comment]
```js
{
  overrides: [
    {
      files: [
        'webpack.config.js'
      ],
      rules: {
        'prefer-module': 'off'
     }
    }
  ]
}
```
